doc/classes/CanvasLayer.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CanvasLayer" inherits="Node" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Canvas drawing layer.
+		A node used for independent rendering of objects within a 2D scene.
 	</brief_description>
 	<description>
-		Canvas drawing layer. [CanvasItem] nodes that are direct or indirect children of a [CanvasLayer] will be drawn in that layer. The layer is a numeric index that defines the draw order. The default 2D scene renders with index 0, so a [CanvasLayer] with index -1 will be drawn below, and one with index 1 will be drawn above. This is very useful for HUDs (in layer 1+ or above), or backgrounds (in layer -1 or below).
-		Embedded [Window]s are placed in layer 1024. CanvasItems in layer 1025 or above appear in front of embedded windows, CanvasItems in layer 1023 or below appear behind embedded windows.
+		[CanvasItem]-derived nodes that are direct or indirect children of a [CanvasLayer] will be drawn in that layer. The layer is a numeric index that defines the draw order. The default 2D scene renders with index [code]0[/code], so a [CanvasLayer] with index [code]-1[/code] will be drawn below, and a [CanvasLayer] with index [code]1[/code] will be drawn above. This order will hold regardless of the [member CanvasItem.z_index] of the nodes within each layer.
+		[CanvasLayer]s can be hidden and they can also optionally follow the viewport. This makes them useful for HUDs like pause menus (on layers [code]1[/code] and higher) or backgrounds (on layers [code]-1[/code] and lower).
```

Comment:
I think HUD and pause menu are 2 different things 🤔

Reply:
Hm, yeah I wasn't trying to be one-for-one here. I went with the logic of "pause menu" will be known by everyone. But now that you're saying this, I agree it's a worse demonstration of the concept.